README.md

```diff
@@ -497,3 +497,54 @@ Some of the options include:
 * `members` - automatically document the members in that Python module.
 * `private-members` - include private functions and methods.
 * `show-inheritance` - show the class inheritance.
+
+# Opentelemetry Instrumentation
+
+IIB is now implemented with Opentelemetry python instrumentation.
+This is done in order to collect detailed information about the behavior and performance of IIB.
+By using a combination of automatic and manual instrumentation, it is possible to capture all relevant information about the application's behavior.
+
+## Environment setup
+The two most important packages to be installed are the `opentelemetry-api` and `opentelemetry-sdk`.
+Please see the requirements.in for more detailed list.
+
+```bash
+pip install virtualenv
+virtualenv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+## Traces
+The application is instrumented to collect traces using OpenTelemetry.
+The traces include information about the duration of each request, as well as any errors that may have occurred.
+
+## Opentelemetry Environment Variables
+These environment variables need to be defined in the `docker-compose.yml` and `podman-compose.yml`
+
+## TODO: Test by adding the below parameters in `podman-compose.yml`
+
+* `OTEL_EXPORTER_OTLP_ENDPOINT` - The endpoint for the OpenTelemetry exporter. By default, this is set to http://localhost:4317.
+* `OTEL_RESOURCE_ATTRIBUTES` - Additional attributes to include in the OpenTelemetry resource. By default, this is set to service.name=flask-app.
```

Comment:
This is a duplicate as well. Could you please remove this?

Reply:
Yup removed